website/docs/docusaurus.config.js

```diff
@@ -162,14 +164,14 @@ module.exports = {
           src: "img/logo-dark.svg",
           srcDark: "img/logo.svg",
         },
-        items: [
-          { to: "/releases", label: "Releases", position: "right" },
-          {
-            href: "https://github.com/codesandbox/sandpack",
-            label: "GitHub",
-            position: "right",
-          },
-        ],
+        // items: [
```

Comment:
you won't go to heaven behaving like this

Reply:
spotted! 😅